Adjustments for core notification and icon changes

bx80 opened the pull request:
### Description:

This PR updates the UI test screenshots affected by the notification and icon updates in core PR [#21077](https://github.com/matomo-org/matomo/pull/21077)

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

Commit:
Adjustments for core notification and icon changes

michalkleiner commented on tests/UI/expected-screenshots/ContainerDashboard_with_entries.png:
Similar issue to the other PR, the plus in the circle icon alignment — looks off centre.

bx80 replied:
Should be fixed with scaled up icons.

michalkleiner commented on tests/UI/expected-screenshots/ContainerTag_confirm_delete_tag_confirmed.png:
Should the chevrons in the lefthand side menu change to the down arrows?

Also the tags in the Triggers column changed which is probably related to the previous change to the links, but it looks weird. Perhaps for links within tags like this we need extra adjustments?

bx80 replied:
They've since been changed back to chevrons.

michalkleiner commented on tests/UI/expected-screenshots/ContainerTag_create_advanced_firetrigger_popup.png:
Should any of the icons in the left column change as well? Some of them are quite bold and not in line with the change overall.

bx80 replied:
The remaining old icons have now all been updated to match the new ones.

michalkleiner commented:
The lefthand menu and the plus in circle icon affect majority of the screenshot changes.

Commit:
Various notification, style and icon improvements

- Temporarily updated submodule references for testing
- Updated the 25 outstanding icons used with core, both with .svg files and updated font files
- icon-done now uses the same thin tick icon as icon-ok
- icon-plus-square and icon-minus-square now have squares again
- New icons icon-add1 and icon-remove have added
- New icons icon-chevron-right and icon-chevron-left have been added
- Updated mobile down menu, dashboard settings, site selector, fieldExandableSelect and reporting menu to use a chevron as before
- The Morpheus UI demo icons section has been tidied to only show the new icon set
- Fixed link colour on marketplace metadata sidebar
- Fixed colour on marketplace plugin tile more link
- Fixed privacy manager opt out non-standard notification
- Increased the size of the warning icon on user permissions menu
- Adjusted help notification background colour
- Reverted three UI tests from a previous commit which were not related to style changes

Commit:
Update UI screenshots after font change

sgiehl committed:
updates expected UI test files

michalkleiner commented on tests/UI/expected-screenshots/ContainerVersion_debug_version_enable.png:
The new screenshot has one more row revision which was an intermittent test failure already seen before, so I guess we'd see one row less on occasions as well.

Probably needs a few more runs to arrive at the same number of rows as in the old screenshot or further investigation into what's causing this test failure? Could be that it needs some more waits in a few places etc. (hopefully just as simple as that, but can't be certain).

michalkleiner commented on tests/UI/expected-screenshots/ContainerVersion_import_version_confirmed.png:
Again different number of rows

sgiehl committed:
updates expected UI test files
